### ISSUE TITLE
Timers missing for Enemy Pet Bar in new version

### ISSUE THREAD

**Issue by SirArioch** [Jaliborc/PetTracker#148]
**Running Softwar (issues missing this information will be deleted):**
 - Addon version:  8.3.1

 **Have you read the changelog? (please don't waste our time)**
YES

**Describe the bug*
Timers missing for Enemy Pet Bar in new version

**To Reproduce**
Do a pet battle

**Expected behaviour**
Timers were present for enemy pets prior to this update.

**Screenshots**
No timer on enemy "Soothe" and "Cleansing Rain" though they are grayed out.
![image](https://user-images.githubusercontent.com/61395385/75122747-beb58f00-5655-11ea-9402-3c9b14bdce54.png)

**Comment by Youzerfriendly** [comment]
Motion Seconded. Took me a while to find out where the problem was, but I'm pretty sure this is it.

**Comment by TurduckenMcNugget** [comment]
Thirded. Would very much like to see the return of the numbers for tracking enemy cooldowns.

**Comment by morticianoire** [comment]
Yup same here they’ve gone since update :-/

**Comment by SirArioch** [comment]
Thanks Bloodleh, this fixed worked perfectly!!

**Comment by Youzerfriendly** [comment]
Worked like a charm! Thanks for that my friend!

**Comment by Jaliborc** [comment]
@Bloodleh That must be it.

**Comment by morticianoire** [comment]
Since new update still not getting timers

**Comment by morticianoire** [comment]
not getting the boxes eithe rof opponents attack list so unsure whats happened

**Comment by morticianoire** [comment]
![image](https://user-images.githubusercontent.com/55747490/76446760-6bff0580-63bf-11ea-98bd-be380abba495.png)

**Comment by hdidier** [comment]
I haven't seen it stated this way, so just to add... I am seeing the timers on wild battles, but not on Trainer or Epic Pet battles.  I'm also not seeing it on PVP, but I don't know if that's by design or not as I did not PVP prior to this week.
Example of missing trainer is Farmer Nishi

**Comment by morticianoire** [comment]
> I haven't seen it stated this way, so just to add... I am seeing the timers on wild battles, but not on Trainer or Epic Pet battles. I'm also not seeing it on PVP, but I don't know if that's by design or not as I did not PVP prior to this week.
> Example of missing trainer is Farmer Nishi

Yes you cld see the timers and stuff in pvp it’s gone again now which is frustrating given it’s pet battle week

**Comment by BenBelatrix** [comment]
I have the same problem - only in PVP pet battles... Very frustrating!
